### PR TITLE
WebGPU API Docs

### PR DESCRIPTION
**Pull request by irvinebroque**

### CODE REVIEW

File: content/workers/runtime-apis/webgpu.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+```
+compatibility_flags = ["experimental", "webgpu"]
+```
+
+The following subset of the WebGPU API is available from within Durable Objects:
+
+| API | Supported? | Notes |
+|-----|------------|-------|
+| [`API`](name) | ✅ | |
```
Comment (edevil) [inline content/workers/runtime-apis/webgpu.md]: This is the list: 

GPUAdapterInfo.vendor
GPUAdapterInfo.architecture
GPUAdapterInfo.device
GPUAdapterInfo.description
GPUAdapter.requestDevice
GPUAdapter.requestAdapterInfo
GPUAdapter.features
GPUAdapter.limits
GPUBindGroupLayout
GPUBindGroup
GPUBuffer.getMappedRange
GPUBuffer.unmap
GPUBuffer.destroy
GPUBuffer.mapAsync
GPUBuffer.size
GPUBuffer.usage
GPUBuffer.mapState
GPUCommandBuffer
GPUCommandEncoder.label
GPUCommandEncoder.beginComputePass
GPUCommandEncoder.copyBufferToBuffer
GPUCommandEncoder.finish
GPUComputePassEncoder.setPipeline
GPUComputePassEncoder.setBindGroup
GPUComputePassEncoder.dispatchWorkgroups
GPUComputePassEncoder.end
GPUComputePipeline.getBindGroupLayout
GPUDevice.createBuffer
GPUDevice.createBindGroupLayout
GPUDevice.createBindGroup
GPUDevice.createSampler
GPUDevice.createShaderModule
GPUDevice.createPipelineLayout
GPUDevice.createComputePipeline
GPUDevice.createCommandEncoder
GPUDevice.destroy
GPUDevice.createQuerySet
GPUDevice.pushErrorScope
GPUDevice.popErrorScope
GPUDevice.queue
GPUDevice.lost
GPUDevice.features
GPUDevice.limits
GPUOutOfMemoryError
GPUValidationError
GPUInternalError
GPUDeviceLostInfo.message
GPUDeviceLostInfo.reason
GPUPipelineLayout
GPUQuerySet
GPUQueue.submit
GPUQueue.writeBuffer
GPUSampler
GPUCompilationMessage.message
GPUCompilationMessage.type
GPUCompilationMessage.lineNum
GPUCompilationMessage.linePos
GPUCompilationMessage.offset
GPUCompilationMessage.length
GPUCompilationInfo.messages
GPUShaderModule.getCompilationInfo
GPUSupportedFeatures
GPUSupportedLimits
GPUMapMode
GPUShaderStage
GPUBufferUsage
GPU.requestAdapter

File: content/workers/runtime-apis/webgpu.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+pcx_content_type: configuration
+title: WebGPU
+---
+
+# WebGPU
+
+The [WebGPU API](https://developer.mozilla.org/en-US/docs/Web/API/WebGPU_API), allows you to use the GPU directly from JavaScript.
+
+The WebGPU API is only accessible from within [Durable Objects](/durable-objects/) — it cannot be used from within Workers.
```
Comment (deadlypants1973) [inline content/workers/runtime-apis/webgpu.md]:
```suggestion
The WebGPU API is only accessible from within [Durable Objects](/durable-objects/). The WebGPU API cannot be used from within Workers.
```
If this is true, then we should put this in DO docs and not Workers. https://developers.cloudflare.com/durable-objects/api/

File: content/workers/runtime-apis/webgpu.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+{{<Aside type="note">}}
+The WebGPU API is currently only available in local development. You cannot yet deploy Durable Objects to Cloudflare that rely on the WebGPU API.
+{{</Aside>}}
+
+To try using the WebGPU API in local development, enable the `experimental` and `webgpu` [compatibility flags](/workers/configuration/compatibility-dates/#compatibility-flags) in the [`wrangler.toml` configuration file](/workers/wrangler/configuration/) of your Durable Object.
```
Comment (deadlypants1973) [inline content/workers/runtime-apis/webgpu.md]:
```suggestion
To use the WebGPU API in local development, enable the `experimental` and `webgpu` [compatibility flags](/workers/configuration/compatibility-dates/#compatibility-flags) in the [`wrangler.toml` configuration file](/workers/wrangler/configuration/) of your Durable Object.
```

File: content/workers/runtime-apis/webgpu.md (hunk 2 above)
Comment (irvinebroque) [reply content/workers/runtime-apis/webgpu.md]: I understand that rationale, but think this is both more discoverable here, and it's not part of the actual Durable Objects API — think of it like an API of the Workers runtime that just happens to be only available from a Durable Object (at least today...)

**Comment by irvinebroque** [comment]
Github having issues, closing in favor of https://github.com/cloudflare/cloudflare-docs/pull/10894

Screenshot 2023-09-26 at 5 29 02 PM

**Pull request closed without merging**